src/NuGet.Clients/NuGet.SolutionRestoreManager/VsSolutionRestoreService.cs

```diff
@@ -196,10 +196,16 @@ private static DependencyGraphSpec ToDependencyGraphSpec(ProjectNames projectNam
                     .Any(tfm => tfm.Version >= Version20);
                 var toolFramework = isNetCore20 ? CommonFrameworks.NetCoreApp20 : CommonFrameworks.NetCoreApp10;
 
+                var packagesPath = GetRestoreProjectPath(projectRestoreInfo.TargetFrameworks);
+                var fallbackFolders = GetRestoreFallbackFolders(projectRestoreInfo.TargetFrameworks).ToList();
```

Comment:
there is no real advantage of passing arrays around, so whenever you get a chance, change them to IEnumerable, which makes it easier to unit test as well.

---
In reply to: [121577543](https://github.com/NuGet/NuGet.Client/pull/1479#discussion_r121577543) [](ancestors = 121577543)